code/__DEFINES/medal.dm

```diff
@@ -46,3 +46,24 @@
 #define MEDAL_DETONATE_WELDERBOMB "That was stupid of you."
 #define MEDAL_SUCCUMB "Please just end the pain"
 
+//Specific Greentext medals
+
+#define MEDAL_GREEN_TRAITOR "Greentext: Traitor"
+#define MEDAL_GREEN_BLOODBRO "Greentext: Blood Brother"
+#define MEDAL_GREEN_IAA "Greentext: Internal Affairs"
+#define MEDAL_GREEN_WIZARD "Greentext: Wizard"
+#define MEDAL_GREEN_CHANGELING "Greentext: Changeling"
+#define MEDAL_GREEN_CULT "Greentext: Cultist"
+#define MEDAL_GREEN_CLOCKCULT "Greentext: Clockwork Cultist"
+#define MEDAL_GREEN_INCURSION "Greentext: Incursion"
+#define MEDAL_GREEN_NUCLEAR "Greentext: Nuclear Operative"
+#define MEDAL_GREEN_TERATOMA "Greentext: Teratoma"
```

Comment:
teratoma always greentext medal for it is stupid

Reply:
Nightmare, Teratoma, and Guardian medals have been removed because they always greentext